### PR TITLE
Upgrade Jenkins Agent Python

### PR DESCRIPTION
**Pull request by jtrinh27**
#### What is this PR About?
Upgrade Jenkins Agent Python:

- origin-jenkins-agent-base from 4.7 to 4.9.
- Python 3.8 to Python 3.9.

#### How do we test this?
CI :)

cc: @redhat-cop/day-in-the-life

### CODE REVIEW

File: jenkins-agents/jenkins-agent-python/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,8 @@
-FROM quay.io/openshift/origin-jenkins-agent-base:4.7
+FROM quay.io/openshift/origin-jenkins-agent-base:4.9
```
Comment (garethahealy) [inline jenkins-agents/jenkins-agent-python/Dockerfile:1]: 4.9 isn't released, so I wouldn't suggest using that tag.

Comment (jtrinh27) [reply jenkins-agents/jenkins-agent-python/Dockerfile:1]: Thanks. Where can I go to find the latest release, I've been using https://quay.io/repository/openshift/origin-jenkins-agent-base?tag=latest&tab=tags

Comment (garethahealy) [reply jenkins-agents/jenkins-agent-python/Dockerfile:1]: I'd typically go to:
- https://docs.openshift.com/container-platform

currently, 4.7 is the latest (_4.8 will be out soon_)

Comment (jtrinh27) [reply jenkins-agents/jenkins-agent-python/Dockerfile:1]: thanks. I will close this PR.

**Pull request closed without merging**